provider/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectContainerExtensions.kt

```diff
@@ -79,24 +95,54 @@ class NamedDomainObjectContainerScope<T : Any>(
     fun <U : T> creating(type: KClass<U>) =
         polymorphicDomainObjectContainer().creating(type)
 
+    /**
+     * Provides a property delegate that creates elements of the given [U].
+     */
+    inline fun <reified U : T> creating() =
+        creating(U::class)
+
     /**
      * Provides a property delegate that creates elements of the given [type] with the given [configuration].
      */
     fun <U : T> creating(type: KClass<U>, configuration: U.() -> Unit) =
         polymorphicDomainObjectContainer().creating(type, configuration)
 
+    /**
+     * Provides a property delegate that creates elements of the given [U] with the given [configuration].
+     */
+    inline fun <reified U : T> creating(noinline configuration: U.() -> Unit) =
+        creating(U::class, configuration)
+
     /**
      * Provides a property delegate that gets elements of the given [type].
      */
     fun <U : T> getting(type: KClass<U>) =
         polymorphicDomainObjectContainer().getting(type)
 
+    /**
+     * Provides a property delegate that gets elements of the given [U].
+     */
+    inline
+    fun <reified U : T> getting() =
+        getting(U::class)
+
```

Comment:
:x: Unnecessary empty line.

Reply:
This should be resolved now. Would really love a linter to check this for me.